src/components/fields/DescriptionField.js

```diff
@@ -10,17 +10,17 @@ function DescriptionField(props) {
     return <div />;
   }
 
-  const renderers = {};
+  const renderers = window.renderers || {};
   if (window.renderCodeBlock) {
     renderers.code = window.renderCodeBlock;
   }
 
   return (
     <div id={id} className={pfx("field-description")}>
-      // TODO: children should be in child level
       <ReactMarkdown
         children={description.replace(/<br>/gi, "\n")}
         renderers={{ ...renderers }}
+        transformLinkUri={null}
       />
```

Comment:
This implementation is not good; there's a better way to doing this. Instead of calling the `ReactMarkdown` component directly, pass a Markdown rendering component to RJSF from the API Portal code via a ["rendering prop"](https://reactjs.org/docs/render-props.html). This way, the RJSF does not have to be dependent on the ReactMarkdown library directly; it just received a rendering prop and uses that to render all descriptions. We can configure the component being passed from the API Portal code to use the same Markdown renderer we are using already and have it preconfigured with out custom renderers beforehand (keeping the RJSF code oblivious to that).